treeherder/webapp/api/performance_serializers.py

```diff
@@ -122,6 +132,11 @@ class PerformanceAlertSummarySerializer(serializers.ModelSerializer):
     push_id = serializers.ReadOnlyField()
     created = serializers.ReadOnlyField()
 
+    def update(self, instance, validated_data):
+        instance.timestamp_first_triage()
+        return super(PerformanceAlertSummarySerializer, self).update(instance,
```

Comment:
I'm wondering why you're calling super here rather than just returning the instance?

Reply:
If the `super(...)` does end up staying, could you convert it to the Python 3 syntax? (`super()` with no arguments, see 769cded9a87a85946eafacb8423db9586bb95a90)

Reply:
> I'm wondering why you're calling super here rather than just returning the instance?
> 
> It looks like this is calling the `update` method on the ModelSerializer? If so, doesn't `instance.save()` achieve the same thing (it creates or updates)? https://www.django-rest-framework.org/api-guide/serializers/#saving-instances

I overrode the `update()` method just to timestamp when a user interacted with an alert summary.
But I want to keep the old automatic update, so it syncs the model with new field values.

Reply:
> 
> 
> If the `super(...)` does end up staying, could you convert it to the Python 3 syntax? (`super()` with no arguments, see [769cded](https://github.com/mozilla/treeherder/commit/769cded9a87a85946eafacb8423db9586bb95a90))

Sure thing.

Reply:
> I overrode the `update()` method just to timestamp when a user interacted with an alert summary.
> But I want to keep the old automatic update, so it syncs the model with new field values.

Right, but I think you can achieve that by using `instance.save()` on your new `update` method (per the link I posted) rather than the calling `update` on the base class? :)

Reply:
> Right, but I think you can achieve that by using `instance.save()` on your new `update` method (per the link I posted) rather than the calling `update` on the base class? :)

I'm not sure I follow. Wouldn't that mean manually reassigning all items from `validated_data` back to instance? That's what `serializers.ModelSerializer.update()` automatically does. Reusing it's method keeps the code DRY.

Reply:
Ah, I see what you mean.